### ISSUE TITLE
Info/warning after giving a color vector to scatter

### ISSUE THREAD

**Issue by carranco-sga** [JuliaPlots/Plots.jl#1717]
Greetings,

I recently upgraded to Julia 0.7 and when trying to plot a set of points with each dot a seperate color with colors specified by a color vector, I keep getting the following warning:

```julia-REPL
┌ Info: it is surprising that this function is called - please report a use case as a Plots issue
└ @ Plots /home/carranco.sga/.julia/packages/Plots/ex9On/src/args.jl:1525
```
I managed to isolate the warning to the usage of the `color` keyword argument with a color vector. A working example is the following:

```julia-REPL
#using Colors
#scheme = colormap("Blues", 10)

using Plots

scheme = [:red, :blue, :black]
scatter(1:10, c = scheme, legend = false)
```
I have tried to update and reinstall Plots with no success. The plot is fine, albeit with that warning. The warning occurs even when when specifying the colors by hand.

Thanks in advance.

**Comment by mkborregaard** [comment]
Thanks - the warning is spurious, and has been taken out on current releases - I should backport a fix to 0.7 so it doesn't show up there.

**Comment by carranco-sga** [comment]
Thanks!
